apps/sms/style/sms.css

```diff
@@ -1062,3 +1062,19 @@ html[dir="rtl"] .contact-header aside.pack-end {
 .editable-select-mode [contenteditable] {
   -moz-user-select: text;
 }
+
+#threads-no-messages, .threadlist-is-empty #threads-container {
+  display: none;
+}
+
+#threads-settings-button, .threadlist-is-empty #threads-options-button {
+  display: none !important;
+}
+
+.threadlist-is-empty #threads-settings-button {
+  display: flex !important;
+}
+
+.threadlist-is-empty #threads-no-messages {
+  display: flex;
```

Comment:
@julienw , with 'display: unset', the icon is not correctly placed.But we are able to set it to right position by giving it some margin.But we need to set !important for the margin also.I saw that we used 'flex' in shared/gaiaheader so I implemented it .Do I need to do with 'display: unset' ?